pkg/indexers/apibinding.go

```diff
@@ -74,3 +74,18 @@ func IndexAPIBindingByBoundResources(obj interface{}) ([]string, error) {
 func APIBindingBoundResourceValue(clusterName logicalcluster.Name, group, resource string) string {
 	return fmt.Sprintf("%s|%s.%s", clusterName, resource, group)
 }
+
+const APIBindingsByIdentityHash = "APIBindingByIdentityHash"
+
+func IndexAPIBindingByIdentityHash(obj interface{}) ([]string, error) {
+	apiBinding, ok := obj.(*apisv1alpha1.APIBinding)
+	if !ok {
+		return []string{}, fmt.Errorf("obj %T is not an APIBinding", obj)
+	}
+
+	var ret []string
+	for _, r := range apiBinding.Status.BoundResources {
+		ret = append(ret, r.Schema.IdentityHash)
```

Comment:
If you keep it this way, you'll want to put this into a `sets.String` because multiple BoundResources can come from the same IdentityHash. But I would switch to indexing based on export reference (path + name)

Reply:
That would be multiple GRs defined on the Schema?